src/pages/index.js

```diff
@@ -1,12 +1,27 @@
 import * as React from "react";
 import { DefaultLayout } from "../components/DefaultLayout";
+import { Button } from "../components/Button";
+import { Container } from "../components/Container";
 
 const IndexPage = () => {
   return (
-    <main className="container m-auto">
+    <Container>
       <title>Portfolio page</title>
-      <DefaultLayout></DefaultLayout>
```

Comment:
Looks like you've started this branch _before_ merging the Header changes. Make sure to rebase the branch (get the latest updates) before merging this one.

Reply:
It looks like you looked at the old changes. (because it was too late, but thank you for spending time on it <3) Or I just don't understand what do you mean.

Reply:
My bad, I think I've reviewed the diff (removed lines). Good find!